templates/pharmacies/general_list.html

```diff
@@ -1,8 +1,8 @@
 <a href="{$CURRENT_ACTION}action=edit" onclick="top.restoreSession()" class="btn btn-secondary btn-add" >
 {xlt t='Add a Pharmacy'}</a><br /><br />
-
-<div class="table-responsive">
-	<table class="table table-striped">
+<p>Total pharmacies {xlt t=$totalpages} </p>
```

Comment:
swap this (translate Total Pharmacies and do not translate $totalpages)

Reply:
done

Reply:
done